src/Authenticator/LoginHandler.php

```diff
@@ -77,8 +77,12 @@ public function link($action = null)
      *
      * @return array
      */
-    public function login()
+    public function login(HTTPRequest $request)
     {
+        if ($backURL = $this->service->getBackURL($request)) {
```

Comment:
Can we move all of this to `RealMeSerive.php`?

Reply:
OK makes sense, I reading the docblock for enforceLogin(), I can see what you've added this here

```
     * Enforce login via RealMe. This can be used in controllers to force users to be authenticated via RealMe (not
     * necessarily logged in as a {@link Member}), in the form of:
     * <code>
     * Session::set('RealMeBackURL', '/path/to/the/controller/method');
     * if($service->enforceLogin()) {
```